Gemfile

```diff
@@ -34,6 +34,7 @@ gem 'sprockets-rails', '>= 2.3.3', '< 3'
 gem 'responders', '~> 2.0'
 gem 'roadie-rails', '~> 1.1'
 gem 'x-editable-rails', '~> 1.5.5'
+gem 'foreman-tasks', '>= 0.7.20'
```

Comment:
Seems we'd need some hack in https://github.com/theforeman/foreman-packaging/blob/deb/develop/debian/xenial/foreman/rules#L23. According to https://github.com/bundler/bundler/issues/2595 bundle package always need to get all gems. What if we removed it from vendor/cache and we'd keep it packaged as a separate dependency, probably moved from plugins repo to core?